### ISSUE TITLE
Fix Cannot set properties of undefined in useKLiveRegion

### ISSUE THREAD

**Issue by AlexVelezLl** [learningequality/kolibri-design-system#1028]
<img height="20px" src="https://i.imgur.com/c7hUeb5.jpeg">

❌ **This issue is not open for contribution. Visit <a href="https://learningequality.org/contributing-to-our-open-code-base/" target="_blank">Contributing guidelines</a>** to learn about the contributing process and how to find suitable issues.

<img height="20px" src="https://i.imgur.com/c7hUeb5.jpeg">

## Product
Kolibri

## Actual behavior

We are getting some [sentry errors](https://learningequality.slack.com/archives/C053DUFQRFZ/p1747310037299139) in Kolibri-demo because of useKLiveRegion:

```
<anonymous>(kolibri-design-system/lib/composables/useKLiveRegion/index)
Cannot set properties of undefined (setting 'textContent')
environment: kolibri-demo
```

## Expected behavior

For the scope of this issue, we should, at least, have some defensive checks against this null pointer exception, and skip live region messages if the live region element isn't mounted yet.

And in general, we should investigate how to ensure the live region element is mounted before starting to send messages. If the latter isn't possible, and it's very important not to lose messages, we could explore creating a queue of pending messages and sending them once the live region element is mounted.

**Comment by vtushar06** [comment]
Hi @AlexVelezLl 👋,

I’d like to work on this issue and implement the defensive checks in useKLiveRegion to prevent the textContent assignment error. Let me know if it’s okay for me to take this up — and feel free to assign me if it’s available.

Thanks!
@vtushar06

**Comment by MisRob** [comment]
Hi @vtushar06, thank you. This issue is not open for contribution. To find out how to contribute, visit [Contributing to our open code base](https://learningequality.org/contributing-to-our-open-code-base).
